modules/environment-roles/templates/github_service_unavailable_deploy_policy.json.tpl

```diff
@@ -0,0 +1,16 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "ServiceUnavailableDeploy",
+      "Effect": "Allow",
+      "Action": [
+        "elasticloadbalancing:DescribeLoadBalancers",
```

Comment:
Alphabetical ordering slightly incorrect, `DescribeListeners` should be before `DescribeLoadBalancers`

Reply:
I've reordered them :+1: